strawberry_django_plus/relay.py

```diff
@@ -632,7 +632,8 @@ def from_nodes(
         if total_count is None:
             # Support ORMs that define .count() (e.g. django)
             try:
-                total_count = int(nodes.count())  # type: ignore
```

Comment:
@bellini666 This wouldn't work in async context, nor does enumerating the queryset.

Maybe there is a wrap to do `sync_to_async` somewhere higher in the stack and it's not doing that?